scripts/performance/benchmarks.json

```diff
@@ -136,30 +107,154 @@
     ]
   },
   {
-    "name": "s3.sync.upload",
-    "command": ["s3", "sync", "./test_dir", "s3://bucket/test_dir", "--quiet"],
+    "name": "dynamodb.getitem",
+    "command": ["dynamodb", "get-item", "--table-name", "MyTable", "--key", "file://key.json", "--return-consumed-capacity", "TOTAL"],
```

Comment:
How much does the `file://` usage impact this output as opposed to specifying the key inline? I see we use it in examples, but curious if it's worth separating DDB's performance from that of `file://`? But if it's relatively small compared to the full operation, okay with this.

Reply:
After chatting internally, it's not significant relative to the overall benchmark (and is how we have the DDB examples documented). Okay with it as-is.